js/index.js

```diff
@@ -1,7 +1,117 @@
-// Iteration 1: Names and Input
+//Iteration 1: Names and Input
+
+let driver = "Dioni";
+console.log("The driver's name is "+ driver)
+let hacker2 = "Nate"
+console.log(`The navigator's name is ${hacker2}`)
+
+//Iteration 2: Conditionals
+
+if( driver.length > hacker2.length){
+  console.log(`The driver has the longest name, it has ${driver.length} characters.`)
+}
+else if ( driver.length < hacker2.length){
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+}
+else {
+  console.log(`Wow, you both have equally long names, ${hacker2.length} characters!`)
+}
+
+//Iteration 3: Loops
+
+for (let i = 0; i < driver.length; i++){
+  console.log(driver[i].toUpperCase())
+}
+console.log('\n');
+
+for (let i = driver.length - 1; i >= 0; i--){
+  console.log(driver[i].toUpperCase())
+}
+
+console.log(`\n`);
```

Comment:
for these interactions, you can use

let output = '';
for (let i = 0; i < hacker1.length; i++) {
    output += hacker1[i].toUpperCase() + ' ';
}
console.log(output.trim());

for (let i = hacker1.length - 1; i >= 0; i--) {
    console.log(hacker1[i]);
}